package.json

```diff
@@ -18,6 +19,7 @@
     "lint": "jshint js preprocessing validation",
     "jasmine-tests": "jasmine",
     "test": "npm run lint && npm run jasmine-tests && npm run validate",
-    "validate": "node validation/markets-validator.js"
+    "validate": "node validation/markets-validator.js",
+    "start": "http-server -f index.html"
```

Comment:
I think all web developers are used to starting a project with `npm start` so they have no habits that need changing.  Or are you thinking about people that have contributed already?

Why don't we just push it like this and see if anyone is having problems?